### DISCUSSION TITLE
We're still processing your application. Once this is complete you will have access to the student developer pack offers.

### DISCUSSION THREAD

**Discussion by pagebook1** [community/community#66773] [GitHub Education] [answered]
### Hi everyone,

Hello Guys, just want to ask when do we get the free education offers? I waited for 3 days and still processing. I only get the free pro plan today but the other benefits are still processing. 
![image](https://github.com/community/community/assets/47450220/0a334cdf-ce95-4542-b070-75c552addf70)
![image](https://github.com/community/community/assets/47450220/e59c5e34-97d7-4cc5-9303-0f37da63db70)
![image](https://github.com/community/community/assets/47450220/3086102d-ad7e-465a-b351-f185829e5b4a)

PS: I requested last september 8 2023 and they need some verification so I reverify myself again this sept 11 2023

**Comment by arafajs** [comment]
It will take 48 - 72 hours. Then the benefits will be activated.

**Reply by pagebook1** [reply]
Thank you for your reply sir. but I already waited for 72 hours. then I resubmit my requirements, so will I have to wait 72 hours again ?

**Reply by arafajs** [reply, marked as answer]
Yes. According to the GitHub documentation, the activation process for the GitHub Student Developer Pack usually takes up to 72 hours. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by pagebook1** [reply]
Thanks for your kindly answer sir.

**Reply by Rjcfer** [reply]
i have same problem , since 3 weeks it is normal to ? 
![image](https://github.com/community/community/assets/87280026/39ad9356-8b32-4c93-8fd6-58cb93c71711)

**Reply by pagebook1** [reply]
I got my student developer pack at exactly 3 days.

**Reply by samus-aran** [reply]
Hi @Rjcfer 👋 

I unfortunately don't see an approved status on your account. Check the status of your[ application via this link](https://education.github.com/discount_requests/application) 👈

**Reply by Rjcfer** [reply]
Thank you, I searched without this link, but I could not find this page without your link.

**Reply by superbhanu22** [reply]
Hey its been weeks since my application is not approved is there anything i can do ?

**Reply by Lelichka** [reply]
I sent the application on September 30, received the purple status approved. However, I didn't get the benefit. I only have this inscription(
![Безымянный1](https://github.com/community/community/assets/70440519/0e5d6ca8-8336-4cf9-95cf-d559f2124e17)
![Безымянный](https://github.com/community/community/assets/70440519/1bd78642-3db5-45c5-9d91-da7f504ff59a)

**Reply by XenonyxBlaze** [reply]
Same here. Submitted my request on 1st October. It shows approved but still haven't been able to claim the benefits.

**Reply by hlzswbc** [reply]
Same here. I was approved on Sept 30th and still not able to get the benefits. Wonder if anyone got the benefits already?

**Reply by Abo-ElSeoud** [reply]
Me too,I submitted my request on September 30th, and although it indicated that my request was approved, I still haven't been able to access the benefits.
![image](https://github.com/community/community/assets/92520583/7cdc18b5-9cd9-4f49-933c-82248ef7cd93)

**Reply by Sabrina-Lopez** [reply]
I have also waited 2 weeks to receive the GitHub Student Developer Pack benefits and offers, and I have an approved status for my application.

**Reply by freeformelm** [reply]
@samus-aran Can you check what's wrong with mine? Show's approved and still this error " We're still processing your application. Once this is complete you will have access to the student developer pack offers. "

**Reply by Justintimefordinner** [reply]
same here, I've been waiting almost a year since my first application (I've periodically re-applied). It kinda feels like they don't actually review the applications :/

**Reply by PanTruskawka045** [reply]
I've also applied on 30th September, and since then I'm waiting for my application to be processed "We're still processing your application. Once this is complete you will have access to the student developer pack offers."

**Reply by anthonychonlane** [reply]
Hi, it's been more than 72 hours but I still haven't received the approval.

**Reply by DubAFK** [reply]
Same, I've been waiting for more than a month

**Comment by kaosalrim** [comment]
Same issue here:
![image](https://github.com/community/community/assets/91023252/165bd1cd-e0e7-4ac1-8d33-da0f208917c7)
says approved, but I still can't access the benefits.
![image](https://github.com/community/community/assets/91023252/96075c8a-fba7-4f98-9b28-2eaf98604b44)

**Reply by Ciornenichii** [reply]
Ciornenichii []()

**Reply by ghost** [reply]
same for me 
:(

**Reply by Kamil-Pawlak** [reply]
I have the same problem. I have submitted at monday evening.

**Reply by ghost** [reply]
same I don't know what the problem is either

**Reply by adityab24840** [reply]
i have the same thing showing.

**Reply by ghost** [reply]
For me it's okay

**Reply by Kamil-Pawlak** [reply]
I got the benefits, you just need to wait

**Reply by Abo-ElSeoud** [reply]
How long did you wait?

**Reply by Kamil-Pawlak** [reply]
Around 72hours

**Reply by Xiaobonor** [reply]
same thing Submitted Nov 11

**Reply by ghost** [reply]
> How long did you wait?

~4 days

**Reply by Xiaobonor** [reply]
Will they send email notifications when passed?

**Reply by adityab24840** [reply]
i got the access today a couple of hours ago

**Reply by cadozera** [reply]
Its my second day i will wait max to monday...

**Reply by saikeerthan-14** [reply]
Its my second week, still no update..

**Comment by iSMP14** [comment]
![Screenshot_3](https://github.com/community/community/assets/132416768/ee068a56-e36e-443f-9620-e52b4da4e6f8)
Same issue, i cant claim any benefit, what can i do?

**Reply by iSMP14** [reply]
![Screenshot_4](https://github.com/community/community/assets/132416768/bc22a8b1-61de-43c8-bbe6-74bea29cd6cf)

**Reply by ghost** [reply]
Wait ~3day

**Comment by Tholkappiar** [comment]
Is there any time limit for this student developer pack ? like 10 months or 12 months ?

**Reply by ilplo** [reply]
1 year duration at a time (at least the one I got previously). My last one has been approved on Jan 10th 2023 and expired exactly on Jan 10th 2024

**Reply by mohit0129** [reply]
> Is there any time limit for this student developer pack ? like 10 months or 12 months ?

It lasts for two years as i read somewhere! And if you are still a student, you can request to renew your pack!

**Comment by AllenChristian** [comment]


**Reply by PriyanshuCK-MNIT** [reply]
I think you must upload a **photo of the document** (clicked by a camera). I guess you've uploaded a screenshot. I faced the same issue but it got resolved by uploading a clicked photo of the hard copy.

**Reply by amitmasram** [reply]
Do you think it worked for you? because I have been facing the same problem also almost   I pulled the 8-9 request

**Reply by PriyanshuCK-MNIT** [reply]
I applied last night. Benefits still need to be available for me. It is just showing approved. I'm expecting them to be available within three days.

**Comment by Rizos19** [comment]
Cuando te dan los beneficios de estudiante, ¿te mandan un correo avisandote?

Muchas gracias :)

**Reply by PriyanshuCK-MNIT** [reply]
Sí

**Comment by jeanchongdev** [comment]
Merry Christmas dear community, may you have a nice day this day with your loved ones. 😊

**Comment by zaneatbp** [comment]
I'm facing the same problem with you. I uploaded the student ID yesterday, and it was approved very soon, however there still shows "We're still processing your application. Once this is complete you will have access to the student developer pack offers." on the education page.

**Comment by liruiw** [comment]
I also have this issue where I have been approved on Dec 26th and still not able to access student benefits (such as deployment of private personal website page). Thanks!

**Comment by ilplo** [comment]
One another issue with newer approval process is taking pictures by your iPhone. was unable to activate back camera on the phone, so had to improvise taking picture of eligibility with front end camera without being able to review picture before taking it :) Are there several ones experienced same issue with back camera?

**Reply by Spacexplorer11** [reply]
Yep, couldn't use the back camera on iPhone **or** iPad

**Reply by dakshverma8899** [reply]
not even on android till now. i applied two days ago and it didnt allowed me to use my back cam.

**Comment by jennatnguyen** [comment]
My student developer pack has taken 8 days so far and is still processing, how much longer?

**Comment by kkirilov1** [comment]
Same here. I applied like 9 days ago, but unfortunately it still hasn't been approved.

**Comment by Jackqhr** [comment]
Same here. I applied 3 days ago, but haven't get the benefit

**Comment by Marek21SK** [comment]
I'm already a week on this 
![image](https://github.com/community/community/assets/99325221/292604d4-1137-4cd1-aaaa-f23adfcb6a6e)

**Reply by Marek21SK** [reply]
I found a fix for my problem on GitHub discussions. I was uploading a 800kb photo. I know 1MB is the limit. But then when I uploaded the same pictures but only 300kb everything check fine and I got email back that 3/4 days and I will get acces. And yes after 3 days I get acces. Sorry for my bad english guys.

**Reply by Spacexplorer11** [reply]
> I found a fix for my problem on GitHub discussions. I was uploading a 800kb photo. I know 1MB is the limit. But then when I uploaded the same pictures but only 300kb everything check fine and I got email back that 3/4 days and I will get acces. And yes after 3 days I get acces. Sorry for my bad english guys. 

You can't even upload photos any more, you have to take a picture

**Comment by Linus-f** [comment]
I have the same issue. I reapplied for the student developer pack around 3 weeks ago (using Firefox). The [pack page](https://education.github.com/pack) showed that my application was still processed. The application din't show up on the [application page](https://education.github.com/discount_requests/application). I thought it would show up when my application was processed but nothing happend.

Today I tried applying again using Chrome. Now my application shows up as approved (green) but nothing happend yet.

There are also some other issues I encountered during the application process.

- The progress bar didn't show up on Firefox
- Selecting another Camera on mobile doesn't work

**Reply by Spacexplorer11** [reply]
Couldn't select another camera as well

**Comment by PranavJaju** [comment]
It have been more than 1 month still I am facing the same issue
![image](https://github.com/community/community/assets/100665512/ed40cab2-dbab-424e-8fb9-e73c81d23fbe)

**Reply by PranavJaju** [reply]
It is  ~5 months now still pending, Can anyone help me
![image](https://github.com/community/community/assets/100665512/c5fcf62f-fd07-4414-8194-810550b7b0dd)
 contact github officials

**Reply by rahmatez** [reply]
Is it now approved by GitHub?

**Comment by Tender-sun** [comment]
Have you solved it yet?

**Comment by Nihonielse** [comment]
(I'm a Japanese. So I used DeepL Translate.)

I've been approved for 4 days now and still can't use it. (4 days = 96 hours) 
(Due to the time difference, dates may differ slightly.)
![image](https://github.com/community/community/assets/146614313/3221b6d9-6937-45a4-a4fa-95ec9a0d8c10) 
![image](https://github.com/community/community/assets/146614313/bf9f8e22-4957-4ea4-85f1-16266bd67e18)

**Comment by BalteMD** [comment]


**Reply by ghost** [reply]
You have to wait up to 72 hours. I was also recently approved but haven't received anything yet, unfortunately.

**Comment by xchetan9** [comment]
No Updates From GitHub , It's 6 Days Now .. it shows approved but no benifit processed

**Comment by adambirds** [comment]
I've been waiting over 10 days now. Shocking it takes this long.

**Reply by samus-aran** [reply]
Hi @adambirds 

It looks like you have been granted benefits as you have the PRO badge on your profile and a valid coupon attached to your account. Can you explain what benefits you haven't received?

**Reply by PranavJaju** [reply]
What about me haven't received the benefits and it's 6 months now.... Application is accepted, hence I can't even apply again

**Comment by 03aey** [comment]
Does it have any expiry date? Or we can apply anytime we want

**Comment by ZahirHumphries** [comment]
<img width="1504" alt="Screenshot 2024-08-30 at 3 27 23 PM" src="https://github.com/user-attachments/assets/26ffe9ad-a79d-4dc8-ac6a-fd515135320d">


Still waiting i'm assuming by Monday I should be approved

**Comment by Spacexplorer11** [comment]
As soon as I hit 72 hours, I got the pack, even though the new semester is starting this week. I think it's just up to your document and how it's reviewed. I don't know why it takes so long to get the benefits even after you are "approved".

**Comment by ANDYHKGSWE** [comment]
<img width="1777" alt="Screenshot 2024-09-10 at 16 29 36" src="https://github.com/user-attachments/assets/48265336-4a37-424c-9e0a-89b999fda38c">

I’m still waiting as well since last Friday, the 6th of September…

**Comment by Royce17** [comment]
same here. Can anyone cancel it and then let me re-submit? please.

**Comment by ghost** [comment]


**Reply by ghost** [reply]
When I try to reapply I get this message
![Screenshot 2024-10-03 at 5 48 09 PM](https://github.com/user-attachments/assets/600b9823-bc0d-4d73-a038-caa5ffbc01fa)

**Comment by dakshverma8899** [comment]
Hello sir/mam,
How you doing?
Hope everything is fine with you.

And the thing about the devpack yeh it was approved on the third day as
promised by you. Thank you GitHub team.
Have a good day 🫶 :)
